modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/TargetRequest.java

```diff
@@ -189,57 +183,43 @@ public void start(NHttpClientConnection conn) throws IOException, HttpException
 			}
 
 		}
+                                                            
+        request = new BasicHttpEntityEnclosingRequest(method, path,
+                version != null ? version : HttpVersion.HTTP_1_1);
         
+        BasicHttpEntity entity = new BasicHttpEntity();
         
-  
-                                                            
-        if (hasEntityBody) {
-            request = new BasicHttpEntityEnclosingRequest(method, path,
-                    version != null ? version : HttpVersion.HTTP_1_1);
-
-            BasicHttpEntity entity = new BasicHttpEntity();
-
-
-            boolean forceContentLength = requestMsgCtx.isPropertyTrue(
-                                                                   NhttpConstants.FORCE_HTTP_CONTENT_LENGTH);
-            boolean forceContentLengthCopy = requestMsgCtx.isPropertyTrue(
-                                                                   PassThroughConstants.COPY_CONTENT_LENGTH_FROM_INCOMING);
-                                
-            if (forceContentLength) {
-                entity.setChunked(false);
-                if (forceContentLengthCopy && contentLength > 0) {
-                    entity.setContentLength(contentLength);
-                } 
-            }else{
-             if (contentLength != -1) {
+        boolean forceContentLength = requestMsgCtx.isPropertyTrue(NhttpConstants.FORCE_HTTP_CONTENT_LENGTH);
+        boolean forceContentLengthCopy = requestMsgCtx.isPropertyTrue(PassThroughConstants.COPY_CONTENT_LENGTH_FROM_INCOMING);
+                            
+        if (forceContentLength) {
+            entity.setChunked(false);
+            if (forceContentLengthCopy && contentLength > 0) {
+                entity.setContentLength(contentLength);
+            } 
+        }else{
```

Comment:
Lets format the code

Reply:
Yes, I did additional commit to format the code.